overlord/devicestate/devicestate.go

```diff
@@ -552,6 +552,7 @@ func Remodel(st *state.State, new *asserts.Model) (*state.Change, error) {
 	}
 
 	chg := st.NewChange("remodel", msg)
+	chg.Set("old-model", string(asserts.Encode(current)))
```

Comment:
this is odd, there should be a way to do this without requiring this

Reply:
This is no longer needed.